src/query_bed.py

```diff
@@ -30,15 +30,20 @@ def main() -> None:
     # FIXME: put your code here
 
 
-    table = Table() ## process bed file (input)
+    bed_table = Table() ## process bed file (input)
     for line in args.bed:
-        table.add_line(parse_line(line))
-        print(line)
-    
-    for line in args.query: 
-        print(line)
-        # hvad vil vi i query???
-
+        bed_table.add_line(parse_line(line))
+
+    for line in args.query:
+        value_query = line.split()
+        new_table = bed_table.get_chrom("chr" + value_query[0][-1])
```

Comment:
This is a bit risky. If the name of the chromosome doesn't match the one in the bed file, you shouldn't try to make guesses. Here, you won't even get it right, because you cannot assume that a chromosome is identified by its last digit. If you have `chr12` in hour query, you will pick the last letter, `2` and put `chr` in front of it.